benchmarks/benchmark_throughput.py

```diff
@@ -168,6 +168,13 @@ def run_vllm(
                 ignore_eos=True,
             ))
         end = time.perf_counter()
+    if outputs:
```

Comment:
Such solution will print outputs always, even if we are using synthetic data (without --dataset argument) or running very high --num-prompt. Please consider some new flag for this, like --print-outputs, to avoid generating huge useless logs.